### DISCUSSION TITLE
support smb protocol?

### DISCUSSION THREAD

**Discussion by zhouchang2017** [minio/minio#18811] [Q&A] [answered]
When deploying Minio server on Ubuntu Server, does Minio support the SMB protocol, allowing convenient access as a local disk on devices such as macOS, iPhone, and Windows?

**Comment by klauspost** [comment, marked as answer]
No. And there is no plan to do so.

**Reply by olljanat** [reply]
@klauspost Many years later I wonder if that is still the case? MinIO originally was S3 only storage solution but #16952 added FTP+SFTP and SMB support would be very useful for those who wants modernize their storage systems but are still forced to provide access for legacy applications.

Example https://github.com/macos-fuse-t/go-smb2 looks to be working Go based SMB server implementation.

**Reply by klauspost** [reply]
> is [it] still the case

Yes.
